client/src/app/search-result-details/search-result-details.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component,Input } from '@angular/core';
+
+@Component({
+  selector: 'op-search-result-detail',
+  templateUrl: 'app/search-result-detail/search-result-detail.component.html',
+  styleUrls: [],
```

Comment:
If you don't use the `styleUrls` selector, delete it, don't leave it empty (also, delete the `providers`, `directives` and `pipes` if you don't use them in this PR).